.circleci/config.yml

```diff
@@ -210,7 +210,7 @@ workflows:
       - build-and-test-linux: &build
           matrix:
             parameters:
-              python-version: &python-versions [3.8.9, 3.9.4, 3.10.0, 3.11.0, 3.12.0]
+              python-version: &python-versions [3.9.4, 3.10.0, 3.11.0, 3.12.0, 3.13.0rc3]
```

Comment:
Can we go to 3.13, or still waiting cibuildwheel?

Reply:
They just now merged the 3.13 PR, so imagine release is imminent. https://github.com/pypa/cibuildwheel/pull/2035

Reply:
cibuildwheel [has been released](https://github.com/pypa/cibuildwheel/releases/tag/v2.21.3)

Reply:
Updated to use 3.13 now. Waiting for review. @arcondello @randomir